src/Workspaces/Core/Portable/CodeRefactorings/CodeRefactoringContext.cs

```diff
@@ -99,6 +104,7 @@ internal void Deconstruct(out Document document, out TextSpan span, out Cancella
         }
     }
 
+    [Obsolete]
```

Comment:
Do we have an issue tracking deleting these obsolete interfaces?

Reply:
No, but I'm working on it already.

Reply:
https://devdiv.visualstudio.com/DevDiv/_git/TypeScript-VS/pullrequest/393027